src/components/Dashboard/TodoList.vue

```diff
@@ -705,20 +714,33 @@ export default {
 
     async loadIncorporationApplication (task) {
       const filing = task.task.filing
-      if (filing && filing.header && filing.incorporationApplication) {
+      if (filing && filing.header) {
+        const title = this.nameRequest
+          ? `${this.entityTypeToName(this.entityType)} Incorporation Application - ${this.entityName}`
+          : `${this.entityTypeToName(this.entityType)} Incorporation Application`
+
+        const subtitle = this.nameRequest
+          ? `NR APPROVED - ${this.expiresText(this.nameRequest)}`
+          : 'DRAFT'
+
         const bcolErr = filing.header.paymentStatusCode || null
         const bcolObj = bcolErr && await this.getErrorObj(bcolErr)
 
+        const haveData = Boolean(filing.offices || filing.contactPoint || filing.parties || filing.shareClasses)
+
         this.taskItems.push({
           filingType: FilingTypes.INCORPORATION_APPLICATION,
           id: filing.header.filingId,
-          title: `${this.entityTypeToName(this.entityType)} Incorporation Application - ${this.entityName}`,
+          title,
+          subtitle,
           draftTitle: 'Incorporation Application',
           status: filing.header.status,
           enabled: Boolean(task.enabled),
           order: task.order,
           paymentToken: filing.header.paymentToken || null,
-          bcolErrObj: bcolObj
+          bcolErrObj: bcolObj,
+          isEmptyFiling: !haveData,
```

Comment:
I need to look into this... doesn't appear to be working.

Please don't merge yet. Thx.

Reply:
Fixed in latest commit!